### ISSUE TITLE
Web support?

### ISSUE THREAD

**Issue by StevenH86** [capacitor-community/intercom#21]
Are there any plans to add web support to this plugin? I see some commented out code that suggests web support was started on.

**Comment by stewones** [comment]
yeah web support would make total sense. I'm tracking the 👍  and will add it in case there're lots of interest.

**Comment by maylorsan** [comment]
any updates?

**Comment by NSiggel** [comment]
> any updates? i can help with this web support update
Has anyone worked on a web PR / Fork ? would be interested as well.

**Comment by maylorsan** [comment]
Hi! @NSiggel 

We currently frozen this functionality and using web open source library for using intercom in web with `isPlatform` capacitor check)

I'm rly wanna to change this think
if u wanna we can cooperate and make this feature together)

**Comment by NSiggel** [comment]
> We currently frozen this functionality and using web open source library for using intercom in web with `isPlatform` capacitor check)
What Angular library as you using if I may ask ?

> I'm rly wanna to change this think if u wanna we can cooperate and make this feature together)
So would be totally interested in seeing what we can come up with since I'm facing similar challenges as well.
I've been testing with a fork 
@houseninja/capacitor-intercom
but I don't seem to be able to get a consistent Web Behavior which is acceptable.

I'm also developing with some 'isPlatform' checks, however I'm currently running into a compatibility issue, since my project is in Angular 13, and 2 open source angular libraries 
https://github.com/CaliStyle/ng-intercom (older dependency which won't deploy in CI/CD pipeline) and a fork 
https://github.com/supy-io/ngx-intercom (which skipped over 13 and now forces 14)

Let me know if you have any suggestions.
Would be great to see this functionality get built into the official capacitor-community/intercom package, and anticipating migration work for Cap 5 rolling out soon.

**Comment by eljass** [comment]
We have web implementation forked into this plugin for anyone still interested on this:
https://github.com/Fiksuruoka-fi/capacitor-intercom

**Comment by stewones** [comment]
closing as not planned but feel free to PR.
